src/rover_control/rover_control.hpp

```diff
@@ -0,0 +1,29 @@
+#include <network.hpp>
+#include "drive_controller.hpp"
+
+namespace rc {
+
+class Drive {
+	public:
+		Drive(net::MessageSender& ms);
+
+		void set_interval(int milliseconds);
+		int get_interval();
+		void poll_events();
+
+		void set_drive_mode(DriveController::DriveMode mode);
+		void halt();
+
+		void set_speed(float speed);
+		void set_angle(float angle);
+		void set_movement(float speed, float angle);
+
+	private:
+		int interval = 100;
+		net::MessageSender sender;
```

Comment:
Make the `net::MessageSender` a reference instead. Like `net::MessageSender&`

In the constructor implementation, instead of setting using assignment, use an initializer list.
Like 
```C++
Drive::Drive(net::MessageSender& ms)
    : sender(ms) {
    // Empty for now
}
```